specs/protocol/README.md

```diff
@@ -435,23 +435,23 @@ In order to prevent storing this sensitive data with the message itself, an RFQ
 
 The `privateData` field is ephemeral and **MUST** only be present when the message is initially sent to the intended recipient.
 
 This table enumerates the structure of `PrivateData` 
-| field    | data type                                         | required | description                                                                           |
-| -------- | ------------------------------------------------- | -------- | ------------------------------------------------------------------------------------- |
-| `salt`   | string                                            | Y        | Randomly generated salt used for hashing PrivateData fields                           |
-| `claims` | string[]                                          | N        | an array of claims that fulfill the requirements declared in an [Offering](#offering) |
-| `payin`  | [`PrivatePaymentDetails`](#privatepaymentdetails) | N        | A container for the unhashed `payin.paymentDetails`                                   |
-| `payout` | [`PrivatePaymentDetails`](#privatepaymentdetails) | N        | A container for the unhashed `payout.paymentDetails`                                  |
+| field    | data type                                         | required | description                                                                                                                                                                                                                                                                                            |
+|----------|---------------------------------------------------|----------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `salt`   | string                                            | Y        | Randomly generated salt used for hashing PrivateData fields                                                                                                                                                                                                                                            |
+| `claims` | string[]                                          | N        | an array of claims that fulfill the requirements declared in an [Offering](#offering). If the creator of the RFQ wants to include no claims, then `claims` SHOULD be omitted rather than being an empty list. `claims` must not be an empty list. |
```

Comment:
```suggestion
| `claims` | string[]                                          | N        | an array of claims that fulfill the requirements declared in an [Offering](#offering). If the creator of the RFQ wants to include no claims, then `claims` MUST be omitted rather than being an empty list. `claims` must not be an empty list. |
```

right?

Reply:
and you can remove the last bit as it's redundant 

```suggestion
| `claims` | string[]                                          | N        | an array of claims that fulfill the requirements declared in an [Offering](#offering). If the creator of the RFQ wants to include no claims, then `claims` MUST be omitted rather than being an empty list. |
```